### PR TITLE
add script for generating simple reporting table

### PR DESCRIPTION
**Pull request by mkudlej**
This is just simple script to generate simple reporting table. We don't want to create from this anything complicated. It can be removed if we find better approach.

### CODE REVIEW

File: generate_testing_status_page.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [ "$#" -gt 0 ]; then
+  echo "Script generates empty page for testing status result for all testcases from: api, web, minimal-acceptance, system-integration"
+  echo "Usage:"
+  echo "  1. Run script without any params in root of repository,"
+  echo "  2. store script output to Doc wiki page,"
+  echo "  3. set RST syntax,"
```
Comment (mbukatov) [inline generate_testing_status_page.sh]: This should be "Markdown".

**Review by mbukatov — COMMENTED** [review]
I noticed one typo, otherwise looks good. It's a hack, but it works good enough.

**Review by fbalak — APPROVED** [review]
LGTM

File: generate_testing_status_page.sh (hunk 1 above)
Comment (mbukatov) [reply generate_testing_status_page.sh]: fixed by 83110a9fbd82e40b19d2373b988c41b175c30b3d

**Review by mbukatov — APPROVED** [review]

**Pull request merged**